tox.ini

```
@@ -140,6 +140,7 @@ envlist =
     python-framework_sanic-{py38,pypy37}-sanic{190301,1906,1812,1912,200904,210300,2109,2112,2203},
     python-framework_sanic-{py37,py38,py39,py310,pypy37}-saniclatest,
     python-framework_starlette-{py310,pypy37}-starlette{0014,0015,0019},
+    python-framework_starlette-{py37}-starlette{002001},
```

Add py38 here so we fully test the BUG_COMPLETELY_FIXED conditional:
```suggestion
    python-framework_starlette-{py37,py38}-starlette{002001},
```